src/main/java/net/minecraftforge/registries/RegistryObject.java

```diff
@@ -296,7 +281,7 @@ public ResourceLocation getId()
      * @return the resource key that points to the registry and name of this registry object
      */
     @Nullable
-    public ResourceKey<T> getKey()
+    public ResourceKey<? super T> getKey()
```

Comment:
What is the reason behind this signature change? Won't it cause issues for people who expect a `ResourceKey<T>`? While this probably isn't runtime breaking especially as it actually returns something that is `ResourceKey<T>` won't it potentially require people to update their mods that use this in order to have them compile?

Reply:
I was fiddling with the generics (which turned out to be futile) on the presumption that the `T` of `ResourceKey` was the registry type, and not the stronger object type.

However, that generic is completely erased, so it's totally pointless - I missed reverting this when I undid the changes.